program/tests/namespace_royalty_enforcement.rs

```diff
@@ -0,0 +1,1133 @@
+#![cfg(feature = "test-bpf")]
```

Comment:
Should we remove the basic_royality_enforcement ruleset now or leave it?  Since its kind of a duplicate.

Reply:
I think it's OK to leave for general test cases.